FND/Accounting/AccountingEquity.rdf

```diff
@@ -103,19 +104,16 @@
 	<owl:Class rdf:about="&fibo-fnd-acc-aeq;FinancialAsset">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-oac-own;TangibleAsset"/>
 		<rdfs:label>financial asset</rdfs:label>
-		<skos:definition>asset in the form of stocks, bonds, rights, certificates, bank balances, etc., as distinguished from tangible, physical assets or intangible assets such as intellectual property</skos:definition>
+		<skos:definition>non-physical, tangible asset whose value is derived from a contractual claim, such as bank deposits, bonds, stocks, rights, certificates, and bank balances</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Financial assets are typically more liquid than other tangible assets, such as commodities or real estate. Financial assets may not cover all assets that might be included on a balance sheet, and do not include tangible, physical assets or intangible assets such as intellectual property.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-acc-aeq;Income">
 		<rdfs:subClassOf>
-			<owl:Class>
-				<owl:unionOf rdf:parseType="Collection">
-					<rdf:Description rdf:about="&fibo-fnd-acc-cur;AmountOfMoney">
-					</rdf:Description>
-					<rdf:Description rdf:about="&fibo-fnd-acc-aeq;FinancialAsset">
-					</rdf:Description>
-				</owl:unionOf>
-			</owl:Class>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
+				<owl:allValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
```

Comment:
So there can be many amounts and many date periods? Do they have to match in cardinality? How to marry the amount with the period given that the values are not ordered?

Reply:
So - in a given date period, one could argue that you could receive multiple payments.  I've constrained this to a specific period, but haven't further constrained the number of payments that might come in during that period.